### DISCUSSION TITLE
New UI

### DISCUSSION THREAD

**Discussion by ghost** [strawberrymusicplayer/strawberry#901] [Ideas]
The software is just as feature pack as I need but the UI is outdated.

If you guys have any mockups or have already laid plans for a UI redesign then please update us.

or i can make all ui components and mockups for this software.

**Comment by jonaski** [comment]
Outdated is your opinion. There are no plans for a new UI. You can make a proposal in a PR, but I can't guarantee that I accept it.

**Reply by bhankas** [reply]
The UI isn't what most would call pretty, but it is functional, fairly intuitive and gets the job done. In spirit of 'Don't fix what ain't broken' and 'Function before Form', I'd vote against this one.
